### PR TITLE
Video: Add support for the music description section

### PR DESCRIPTION
**Pull request by techmetx11**

### CODE REVIEW

**Comment by SamantazFox** [comment]
Hello! Thanks for your PR :)

Could you also add support for videos with many songs in them? (e.g: `t3SCffJ0eEg`)

**Comment by techmetx11** [comment]
Sure thing! :)

File: src/invidious/videos/parser.cr [hunk 1]
```diff
@@ -309,6 +309,29 @@ def parse_video_info(video_id : String, player_response : Hash(String, JSON::Any
     end
   end
 
+  # Music section
+
+  music_list = [] of VideoMusic
+  music_desclist = player_response.dig?("engagementPanels", 1, "engagementPanelSectionListRenderer", "content", "structuredDescriptionContentRenderer", "items", 2, "videoDescriptionMusicSectionRenderer", "carouselLockups").try &.as_a
+  music_desclist.try &.each do |music_desc|
```
Comment (SamantazFox) [inline src/invidious/videos/parser.cr:316]:
```suggestion
  music_desclist = player_response.dig?(
    "engagementPanels", 1, "engagementPanelSectionListRenderer",
    "content", "structuredDescriptionContentRenderer", "items", 2,
    "videoDescriptionMusicSectionRenderer", "carouselLockups"
  )
  
  music_desclist.try &.as_a.each do |music_desc|

Comment (SamantazFox) [reply src/invidious/videos/parser.cr:316]: Proposition adds wrapping for clarity, and uses a single `.try`

**Review by SamantazFox — CHANGES_REQUESTED** [review]

**Comment by SamantazFox** [comment]
Regarding the HTML, what do you think about using the section title as the toggle, instead of the "show more/less" button?

image
image

For your information, I used the following for the arrows:
`<i class="icon ion-ios-arrow-up"></i>`
`<i class="icon ion-ios-arrow-down"></i>`

**Comment by techmetx11** [comment]
> Regarding the HTML, what do you think about using the section title as the toggle, instead of the "show more/less" button?
> 
> image image
> 
> For your information, I used the following for the arrows: `<i class="icon ion-ios-arrow-up"></i>` `<i class="icon ion-ios-arrow-down"></i>`
Seems alot more better, Also thanks for your reviews :)

**Review by SamantazFox — APPROVED** [review]

**Comment by SamantazFox** [comment]
Thanks for contributing to invidious :D

**Pull request merged**